src/pieces/memberHandler.ts

```diff
@@ -35,6 +39,21 @@ async function memberUpdate(oldMember: GuildMember | PartialGuildMember, newMemb
 	}
 }
 
+async function memberRemove(member: GuildMember | PartialGuildMember): Promise<void> {
+	if (member.guild.id !== GUILDS.MAIN) return;
```

Comment:
Im not sure if a partial member has a guild attribute. You may want to try and test this by clearing the members cache then leaving a guild.